charts/pega/values-cdh.yaml

```diff
@@ -194,7 +194,27 @@ global:
         type: RollingUpdate
 
       hpa:
-        enabled: true
+        enabled: false
+
+    - name: "RTDG"
+      # Create a background tier for BIX processing.  This tier uses
```

Comment:
please spell out what RTDG stands for and replace BIX in the comments here as well->
> Create a background tier for BIX processing.  This tier uses
> the BIX node type and will not be exposed to the load balancer